core/org.wso2.carbon.core/src/main/java/org/wso2/carbon/core/util/KeyStoreManager.java

```diff
@@ -217,6 +215,7 @@ public Key getPrivateKey(String keyStoreName, String alias) {
      * @return password of the key store
      * @throws Exception Error when reading the registry resource of decrypting the password
      */
+    // TODO: deprecate this method in the future since we are no longer using registry for key store management.
```

Comment:
Let's incorporate this change as well.

Reply:
Resolved with 266088422bbbed98fef56e2450886aa103d0a3d4.
(The usage will be removed with a different PR.)